frontend/lib/src/components/widgets/DataFrame/columns/ChartColumn.ts

```diff
@@ -187,9 +186,7 @@ function BaseChartColumn(
  * This column type is currently read-only.
  */
 export function LineChartColumn(props: BaseColumnProps): BaseColumn {
-  // TODO(lukasmasuch): Change this to use `line` chart and introduce
-  // an AreaChartColumn for the `area` chart.
-  return BaseChartColumn("line_chart", props, "area")
+  return BaseChartColumn("line_chart", props, "line")
```

Comment:
nit: you could think about defining `line_char`/`area_chart` and `line`/`area` as constants in the top of the file and use them here and in the tests.

Reply:
Updated it as suggested 👍